packages/grid/_modules_/grid/models/colDef/gridColType.ts

```diff
@@ -4,5 +4,5 @@ export const GRID_DATE_COLUMN_TYPE = 'date';
 export const GRID_DATETIME_COLUMN_TYPE = 'dateTime';
 export const GRID_BOOLEAN_COLUMN_TYPE = 'boolean';
 
-export type GridNativeColTypes = 'string' | 'number' | 'date' | 'dateTime' | 'boolean';
+export type GridNativeColTypes = 'string' | 'number' | 'date' | 'dateTime' | 'boolean' | 'select';
```

Comment:
Are we sure about the name of the type? I'm asking because of Airtable double type:

<img width="430" alt="Capture d’écran 2021-06-25 à 01 23 48" src="https://user-images.githubusercontent.com/3165635/123344391-0bfc8600-d554-11eb-8f20-f288019c0744.png">

Reply:
Regarding the multiple select - we can set it up by renaming the `select` type to `singleSelect` or keeping it as it is but then the have the multiple one be called `multipleSelect`

Reply:
`singleSelect` would be better. `multipleSelector` is completely different (e.g. needs a "contains" operator).

Reply:
+1 for removing potential confusion

```suggestion
export type GridNativeColTypes = 'string' | 'number' | 'date' | 'dateTime' | 'boolean' | 'singleSelect';
```